app/views/media/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="row">
+  <div class="col-lg-12">
+    <h1><%= @medium.name %> <small>Directed by: <%= @medium.creator %></small></h1>
+    <h4>Ranked:
+      <% if @medium.upvotes.nil? %>
+        0
+      <% else %>
+        <%= @medium.upvotes %>
+      <% end %>
```

Comment:
If you were to set a default value for `upvotes` of 0 in your model (see earlier comment), then you would be able to reduce these five lines of code in one :+1: 

`Ranked: <%= @medium.upvotes %>`